_docs/setup/install-kubernetes.md

```diff
@@ -73,59 +71,29 @@ default namespace. They can be modified for deployment in a different namespace.
-   [RBAC (Role-Based Access Control)](https://kubernetes.io/docs/admin/authorization/rbac/) enabled:
+1. Install Istio's core components.  Choose one of the two _**mutually exclusive**_ options below:
 
-   ```bash
-   kubectl api-versions | grep rbac
-   ```
-   * If the command displays an error, or does not display anything, it means the cluster does not support RBAC, and you can proceed to step 5 below.
-   
-   * If the command displays 'beta' version, or both 'alpha' and 'beta', please apply istio-rbac-beta.yaml configuration as show below:
-   
-   *(Note: If you deploy Istio in another namespace than the `default` namespace, replace the `namespace: default` line 
-   in all ClusterRoleBinding resources with the actual namespace.)*
-   
-   ```bash
-   kubectl apply -f install/kubernetes/istio-rbac-beta.yaml
-   ```
-   If you get an error
-   ```
-   Error from server (Forbidden): error when creating "install/kubernetes/istio-rbac-beta.yaml": clusterroles.rbac.authorization.k8s.io "istio-pilot" is forbidden: attempt to grant extra privileges: [{[*] [istio.io] [istioconfigs] [] []} {[*] [istio.io] [istioconfigs.istio.io] [] []} {[*] [extensions] [thirdpartyresources] [] []} {[*] [extensions] [thirdpartyresources.extensions] [] []} {[*] [extensions] [ingresses] [] []} {[*] [] [configmaps] [] []} {[*] [] [endpoints] [] []} {[*] [] [pods] [] []} {[*] [] [services] [] []}] user=&{user@example.org [...]
-   ```
-   You need to add the following: (replace the name with your own)
-   ```
-   kubectl create clusterrolebinding myname-cluster-admin-binding --clusterrole=cluster-admin --user=myname@example.org
-   ```
-   
-   * If the command displays only 'alpha' version, please apply istio-rbac-alpha.yaml configuration:
-   
-   *(Note: If you deploy Istio in another namespace than the `default` namespace, replace the `namespace: default` line 
-      in all ClusterRoleBinding resources with the actual namespace.)*
-      
-   ```bash
-   kubectl apply -f install/kubernetes/istio-rbac-alpha.yaml
-   ```
-
-1. Install Istio's core components .
-   There are two mutually exclusive options at this stage:
-
-    * Install Istio without enabling [Istio Auth]({{home}}/docs/concepts/network-and-auth/auth.html) feature:
+    * Install Istio without enabling [authentication]({{home}}/docs/concepts/network-and-auth/auth.html) between sidecars with [mutual TLS authentication](https://en.wikipedia.org/wiki/Mutual_authentication).
+    We recommend this option for clusters with existing applications, to allow new applications to communicate with the old applications,
+    and for clusters that use [liveliness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/), headless services, or statefulsets.
 
       ```bash
       kubectl apply -f install/kubernetes/istio.yaml
       ```
-   
-      This command will install Pilot, Mixer, Ingress-Controller, Egress-Controller core components.
```

Comment:
I think it is valuable to list what will be installed without auth, similarly like you listed below for the install cmd with auth.

Reply:
We install the CA component with or without auth. To minimize disruption. And the CA component does not run until used. Which is why she removed this piece.